data/json/construction.json

```diff
@@ -2309,5 +2309,23 @@
     "pre_note": "Can be deconstructed without tools.",
     "pre_terrain": "t_pit_shallow",
     "post_terrain": "f_firering"
+  },
+  {
+    "type": "construction",
+    "description": "Build Rammed Earth Wall",
+    "//": "No wood required! Cleans up dirt quick! Frame is used only to shape the earth, and can be reused afterward.",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 2 ], [ "survival", 2 ] ],
+    "time": 180,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "byproducts": [ { "item": "frame_wood_light", "count": 1 }, { "item": "stick_long", "count": 1 } ],
```

Comment:
It might be better to specify recoverable components as tools, not as components and byproducts, so that their damage state, for example, are preserved.

Reply:
Working on that now. Didn't realise that was an option for constructions, because they all use tool qualities for most stuff now.